tests/int/test-just_git_airgap_repo.bsh

```diff
@@ -0,0 +1,623 @@
+#!/usr/bin/env bash
+
+if [ -z ${VSI_COMMON_DIR+set} ]; then
+  VSI_COMMON_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.."; pwd)"
+fi
+
+source "${VSI_COMMON_DIR}/tests/testlib.bsh"
+command -v "${GIT-git}" &> /dev/null && source "${VSI_COMMON_DIR}/linux/just_git_airgap_repo.bsh"
+source "${VSI_COMMON_DIR}/linux/requirements.bsh"
+
+# These tests chain together, so no point in continuing after one breaks
+TESTLIB_STOP_AFTER_FAILS=1
+unset TESTLIB_SKIP_TESTS TESTLIB_RUN_SINGLE_TEST
+
+if ! git lfs &> /dev/null; then
+  TESTLIB_SKIP_TESTS='.*'
+fi
+
+function setup_variables()
+{
+  # Easier way to take care of committers on vanilla install
+  export GIT_COMMITTER_NAME='Foo Bar'
+  export GIT_AUTHOR_NAME='Foo Bar'
+  export GIT_COMMITTER_EMAIL='foo.bar@example.com'
+  export GIT_AUTHOR_EMAIL='foo.bar@example.com'
+
+  VSI_COMMON_URL="${TRASHDIR}/git/VisionSystemsInc/vsi_common.git" # bare repo
+  RECIPES_URL="${TRASHDIR}/git/VisionSystemsInc/docker_recipes.git" # bare repo
+  PRETEND_URL="${TRASHDIR}/git/pretend_repo.git" # bare repo
+  BUILD_REPO="${TRASHDIR}/build"
+  PREP_DIR="${TRASHDIR}/pretend_prep"
+  TRANSFER_DIR="${TRASHDIR}/transfer"
+  AIRGAP_MIRROR_DIR="${TRASHDIR}/airgap/mirror"
+  AIRGAP_CLONE_DIR="${TRASHDIR}/airgap/clone"
+
+  mkdir -p "${VSI_COMMON_URL}"
+  mkdir -p "${RECIPES_URL}"
+  mkdir -p "${PRETEND_URL}"
+  mkdir -p "${BUILD_REPO}"
+  mkdir -p "${TRANSFER_DIR}"
+  mkdir -p "${AIRGAP_CLONE_DIR}"
+}
+
+command -v "${GIT-git}" &> /dev/null || skip_next_test
```

Comment:
This can probably be combined with the git lfs test on line 15, instead of being put in front of each individual test. Same true in the git_mirror int test, I just didn't think of it when I wrote it originally.